hathor/websocket/factory.py

```diff
@@ -105,23 +116,18 @@ def _setup_rate_limit(self):
             self.rate_limiter.set_limit(control_type, config['max_hits'], config['hits_window_seconds'])
             self.buffer_deques[control_type] = deque(maxlen=config['buffer_size'])
 
-    def _schedule_and_send_metric(self):
+    def _send_metrics(self):
         """ Send dashboard metric to websocket and schedule next message
```

Comment:
Update the docstring. We are not scheduling the next message anymore.

Reply:
Done.